.github/docker/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: '3'
+
+services:
+  master:
+    image: minglirui/spark-base:2.4.4
+    command: master
+    ports:
+      - 8080:8080
+      - 7077:7077
+    environment:
+      - SNOWFLAKE_TEST_CONFIG=/Users/ec2-user/snowflake.travis.json
+
+  worker_1:
+    image: minglirui/spark-base:2.4.4
+    command: worker spark://master:7077
+    depends_on:
+      - master
+    ports:
+      - 8081:8081
+    environment:
+      - SPARK_WORKER_WEBUI_PORT=8081
+      - SPARK_WORKER_MEMORY=2g
+      - SPARK_WORKER_CORES=2
+      - SNOWFLAKE_TEST_CONFIG=/Users/ec2-user/snowflake.travis.json
+
+  worker_2:
+    image: minglirui/spark-base:2.4.4
+    command: worker spark://master:7077
+    depends_on:
+      - master
+    ports:
+      - 8082:8082
+    environment:
+      - SPARK_WORKER_WEBUI_PORT=8082
+      - SPARK_WORKER_MEMORY=2g
+      - SPARK_WORKER_CORES=2
+      - SNOWFLAKE_TEST_CONFIG=/Users/ec2-user/snowflake.travis.json
+
+  testdriver:
+    image: minglirui/spark-base:2.4.4
+    command: spark-submit \
+      --jars /Users/ec2-user/spark-snowflake_2.11-2.7.0-spark_2.4.jar,/Users/ec2-user/snowflake-jdbc-3.12.2.jar \
+      --master spark://master:7077 --deploy-mode client \
+      --class  net.snowflake.spark.snowflake.ArrowPerfTest  \
+      /Users/ec2-user/spark-snowflake-perf_2.11-2.7.0-spark_2.4.jar
+    depends_on:
+      - master
+    environment:
+      - SNOWFLAKE_TEST_CONFIG=/Users/ec2-user/snowflake.travis.json
+      - SPARK_CONN_ENV_SFCOMPRESS=on
+      - SPARK_CONN_ENV_SFSSL=on
+      - SPARK_CONN_ENV_RUNOPTION=snowflake-stage
+      - SPARK_CONN_ENV_SFDATABASE=testdb_spark
+      - SPARK_CONN_ENV_SFSCHEMA=spark_test
+      - SPARK_CONN_ENV_SFWAREHOUSE=testwh_spark
+      - SPARK_CONN_ENV_SFUSER=test_spark
+      - SPARK_CONN_ENV_PEM_PRIVATE_KEY=$SPARK_CONN_ENV_PEM_PRIVATE_KEY
+      - SPARK_CONN_ENV_SFURL=sfctest0.snowflakecomputing.com
```

Comment:
I think we need hide all login info not only the private key.

Reply:
got it. I just want to make it work first. long team, I want to pass the encrypted json file and decrypt it in side the container.